lib/modules/datasource/go/releases-direct.spec.ts

```diff
@@ -11,6 +11,16 @@ const datasource = new GoDirectDatasource();
 const getDatasourceSpy = jest.spyOn(BaseGoDatasource, 'getDatasource');
 const hostRules = mocked(_hostRules);
 
+const getGithubTags = jest.fn();
+jest.mock('../github-tags', function () {
+  const { GithubTagsDatasource: Orig } = jest.requireActual('../github-tags');
+  const Mocked = jest.fn().mockImplementation(() => ({
+    getReleases: (...args) => getGithubTags.call(null, ...args),
```

Comment:
```suggestion
    getReleases: (...args) => getGithubTags(...args),
```
no need for call, why not simply assign?

```suggestion
    getReleases: getGithubTags,
```